.github/workflows/main.yml

```diff
@@ -104,7 +104,7 @@ jobs:
       - name: install examples dependencies
         run: sudo apt-get install libsdl2-dev
       - name: configure examples
-        run: cmake -DCMAKE_PREFIX_PATH=$(pwd)/build/release/third_party/install -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWERROR=OFF -Bexamples/build -Hexamples
+        run: cmake -DCMAKE_PREFIX_PATH=$(pwd)/install;$(pwd)/build/release/third_party/install -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWERROR=OFF -Bexamples/build -Hexamples
```

Comment:
I would use double quotes, because otherwise I am not sure how the `;` is interpreted.

Reply:
Yep, seems like that was it :blush:.

Reply:
Thanks!